test/connectors/GuidanceConnectorSpec.scala

```diff
@@ -18,31 +18,42 @@ package connectors
 
 import base.BaseSpec
 import uk.gov.hmrc.http.HeaderCarrier
-
+import mocks.{MockAppConfig, MockHttpClient}
 import scala.concurrent.ExecutionContext.Implicits.global
-
-class GuidanceConnectorSpec extends BaseSpec {
+import play.api.test.{DefaultAwaitTimeout, FutureAwaits}
+import play.api.libs.json.Json
+import models.ocelot.Process
+import scala.concurrent.Future
+
+class GuidanceConnectorSpec extends BaseSpec with MockHttpClient {
```

Comment:
Both class and test trait extend MockHttpClient. Can probably just define on trait.